<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>LoginScreen</class>
  <widget class="QWidget" name="LoginScreen">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>800</height>
      </rect>
    </property>
    <property name="styleSheet">
      <string notr="true">QWidget#LoginScreen {
    background-color: #303030;
}
QLabel {
    color: white;
    font-size: 48pt;
}
QLineEdit {
    background-color: #404040;
    color: white;
    border: 2px solid #505050;
    border-radius: 10px;
    padding: 15px;
    font-size: 48pt;
    margin: 10px;
    min-height: 80px;
}
QPushButton {
    background-color: #404040;
    color: white;
    border: 2px solid #505050;
    border-radius: 10px;
    padding: 10px;
    font-size: 48pt;
    min-width: 300px;
    min-height: 120px;
}
QPushButton:hover {
    background-color: #505050;
    border-color: #606060;
}
QPushButton:pressed {
    background-color: #606060;
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
        <number>40</number>
      </property>
      <property name="leftMargin">
        <number>80</number>
      </property>
      <property name="topMargin">
        <number>80</number>
      </property>
      <property name="rightMargin">
        <number>80</number>
      </property>
      <property name="bottomMargin">
        <number>80</number>
      </property>
      <item>
        <layout class="QHBoxLayout" name="logoLayout">
         <item>
          <spacer name="logoLeftSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="logoLabel">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/Images/SandDriveLogo.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>450</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1200</width>
             <height>450</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="logoRightSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
      </item>
      <item>
        <widget class="QLabel" name="usernameLabel">
          <property name="text">
            <string>Username</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="passwordEdit">
          <property name="echoMode">
            <enum>QLineEdit::Password</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" name="buttonLayout">
          <item>
            <widget class="QPushButton" name="backButton">
              <property name="text"><string>Back</string></property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="loginButton">
              <property name="text"><string>Login</string></property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
